samples/test-samples.cmake

```diff
@@ -38,9 +38,18 @@ foreach (SAMPLE data-sealing)
     COMMAND ${CMAKE_COMMAND} --build ${SOURCE_DIR}/${SAMPLE}
     WORKING_DIRECTORY ${SAMPLE_BUILD_DIR})
 
-  execute_process(
-    COMMAND ${CMAKE_COMMAND} --build ${SAMPLE_BUILD_DIR} --target run
-    RESULT_VARIABLE TEST_RESULT)
+  if (NOT DEFINED $ENV{OE_SIMULATION})
+    execute_process(
+      COMMAND ${CMAKE_COMMAND} --build ${SAMPLE_BUILD_DIR} --target run
+      RESULT_VARIABLE TEST_RESULT)
+  endif ()
+
+  if (${SAMPLE} MATCHES "(file-encryptor|helloworld)")
+    # Simulation tests
```

Comment:
This comment doesn't add much value. Maybe, "Of all the current samples, only `file-encryptor` and `helloworld` also work under simulation, so we test that additional scenario here."

Otherwise it's not that clear why we're doing this.

Reply:
Sorry about it, I folded in you comments and added you as co-author.